test/mocks/mappings/addresses.json

```diff
@@ -49,6 +49,26 @@
           ]
         }
       }
+    },
+    {
+      "scenarioName": "Address",
+      "requiredScenarioState": "Started",
+      "newScenarioState": "AddressResponses",
+      "request": {
+        "method": "GET",
+        "url": "/ordnance?&postcode=XXX_XX&key=secret",
+        "headers": {
+          "x-scenario-id": {
+            "equalTo": "address-success"
+          }
+        }
+      },
+      "response": {
+        "status": 404,
```

Comment:
Is this what comes back from the OS API if no results are found?

Reply:
@merlinc No, it returns 200 but an empty results object. 

However, in the Friday AddressCRI catchup last week there was talk of the backend api returning 404 if we cant find an address. So the preparation was for that here.

Reply:
👍